__tests__/app.test.js

```diff
@@ -340,3 +340,34 @@ describe("PATCH /api/articles/article_id:", () => {
       });
   });
 });
+
+describe("DELETE /api/comments/comment_id:", () => {
+  test("204: Deletes the comment from the database", () => {
+    return request(app)
+      .delete("/api/comments/3")
+      .expect(204)
+      .then(() => {
+        return db
+          .query(`SELECT * FROM comments WHERE comment_id = 3`)
+          .then(({ rows }) => {
+            expect(rows.length).toBe(0);
+          });
+      });
+  });
+  test("404: gives a 404 status when the comment id is valid but cannot be found", () => {
+    return request(app)
+      .delete("/api/comments/999")
+      .expect(404)
+      .then(({ body: { msg } }) => {
+        expect(msg).toBe("comment not found");
+      });
+  });
+  test("400: gives a bad request when the comment id is invalid", () => {
+    return request(app)
+      .delete("/api/comments/fghj")
```

Comment:
if I'm being picky id make the string clearer - not-a-number or invalidId or something 😄 

great testing though!